tests/suites/0_stateless/05_ddl/05_0000_ddl_create_tables.sql

```diff
@@ -62,7 +62,7 @@ create table db2.test6(id Int8, created timestamp  DEFAULT today() + 3);
 
 
 SELECT '====CREATE ALL DATA TYPE TABLE====';
-create table db2.test7(tiny TINYINT, tiny_unsigned TINYINT UNSIGNED, smallint SMALLINT, smallint_unsigned SMALLINT UNSIGNED, int INT, int_unsigned INT UNSIGNED, bigint BIGINT, bigint_unsigned BIGINT UNSIGNED,float FLOAT, double DOUBLE, date DATE, datetime DATETIME, ts TIMESTAMP, str VARCHAR default '3', bool BOOLEAN, array ARRAY, obj OBJECT, variant VARIANT);
+create table db2.test7(tiny TINYINT, tiny_unsigned TINYINT UNSIGNED, smallint SMALLINT, smallint_unsigned SMALLINT UNSIGNED, int INT, int_unsigned INT UNSIGNED, bigint BIGINT, bigint_unsigned BIGINT UNSIGNED,float FLOAT, double DOUBLE, date DATE, datetime DATETIME, ts TIMESTAMP, str VARCHAR default '3', bool BOOLEAN, arr ARRAY, obj OBJECT, variant VARIANT);
```

Comment:
I modified the test case here because "array" is a reserved ident for postgresql.

Reply:
Oh, it's a reserved ident for postgres only when following an `AS`.